games-simulation/freerct/freerct-0.2~git.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Open source recreation of RollerCoaster Tycoon 1 and 2"
+DESCRIPTION="FreeRCT is a free and open-source theme park simulation game \
+that aims to capture the look, feel, and gameplay of the renowned classic \
+games RollerCoaster Tycoon 1 and 2 with original assets.
+
+The game is still in an early alpha state, but it is already playable and \
+offers a variety of features."
+HOMEPAGE="https://freerct.net/"
+COPYRIGHT="2011-2022 FreeRCT team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="f123d01511682f7e2fd20ce52e6779c35086153a"
+SOURCE_URI="https://github.com/win8linux/FreeRCT/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="b9f752054d30520b2270f7b4035eea7ce04c7a3ccdc6a093af68781fd4f8b1b2"
+SOURCE_DIR="FreeRCT-$srcGitRev"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	freerct$secondaryArchSuffix = $portVersion
+	app:freerct = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+    lib:libpng16$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libSDL2_ttf_2.0$secondaryArchSuffix
+    lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+    devel:libpng16$secondaryArchSuffix
+	devel:libSDL2_2.0$secondaryArchSuffix
+	devel:libSDL2_ttf_2.0$secondaryArchSuffix
+    devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake -DRELEASE=ON -DCMAKE_BUILD_TYPE=Release -DASAN=OFF -DCMAKE_INSTALL_PREFIX=$appsDir/FreeRCT ..
+	make $jobArgs
```

Comment:
You can combine these with:
```
cmake -B build -S . -DRELEASE=ON -DCMAKE_BUILD_TYPE=Release \
	-DASAN=OFF -DCMAKE_INSTALL_PREFIX=$appsDir/FreeRCT
make -C build $jobArgs
```
I'm checking if we could use $cmakeDirArgs here (will get back on that)